config/locales/formats/associations/en.yml

```diff
@@ -0,0 +1,11 @@
+en:
```

Comment:
To avoid having lots of `en.yml` files, you can put `config.i18n.load_path += Dir[Rails.root.join('config', 'locales', 'en', '*.yml')]` in your `application.rb`. Then you can put everything in `/locales/en/*.yml`.

Reply:
Good idea - done.